contributing/getting-started-quick.md

```diff
@@ -27,9 +27,9 @@ This Quick Start guide is the TL;DR version of the longer [end-to-end guide](get
 
 * Install [NodeJS](https://nodejs.org/)
 
-* Install [yarn](https://yarnpkg.com/en/docs/install)
+* Install [npm](https://www.npmjs.com/get-npm)`
 
-* Install Gulp by running `yarn global add gulp`
+* Install Gulp by running `sudo npm install -g gulp`
```

Comment:
Same comment re: `sudo`. Many of our developers are on Mac OS and probably don't need to use it.

Reply:
Same